lib/kitchen/driver/openstack.rb

```diff
@@ -82,23 +81,22 @@ def create(state)
             config[:server_name] = default_name
           end
         end
-        config[:disable_ssl_validation] && disable_ssl_validation
+        disable_ssl_validation if config[:disable_ssl_validation]
```

Comment:
Where would this option ever get set?  I don't see a configuration option in your driver or the base driver for :disable_ssl_validation